examples/single_page_with_id.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+require_once(__DIR__ . '/autoload.php');
+
+use Usabilla\API\Client\UsabillaClient;
+
+if ($argc !== 5) {
+    die('Usage: ' . basename(__FILE__) . ' [command] [id] [access-key] [secret-key]' . PHP_EOL);
+}
+
+list($commandName, $id, $accessKey, $secretKey, $id) = array_slice($argv, 1);
```

Comment:
The id is mapped from the second argument, not the fourth, please remove the last one (also executing this scripts gives an `Undefined offset` notice)